cloud-data-protection/src/services/authService.tsx

```diff
@@ -0,0 +1,23 @@
+import store from '../stores/Store';
+import {http} from "../common/http";
+import {login, logout} from "../features/userSlice";
+import {ServiceBase} from "./base/serviceBase";
+import {AxiosResponse, CancelToken} from "axios";
+import {LoginInput} from "./input/loginInput";
+import {LoginResult} from "./result/loginResult";
+
+export class AuthService extends ServiceBase {
+    public async login(input: LoginInput, cancelToken?: CancelToken) {
+        await http.post('/Authentication/Authenticate', input, { cancelToken: cancelToken })
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
        await http.post('/Authentication/Authenticate', input, { cancelToken })
```